client/components/Main/ListPanel.tsx

```diff
@@ -375,7 +377,14 @@ export class ListPanel extends React.Component<IProps, IState> {
 
                             let listGroupProps: {[key: string]: any} = {
                                 name: group.date,
-                                items: group.events,
+
+                                /**
+                                 * lodash `get` is used, so we can access the sort method from lodash
+                                 * and pass a scoring function to it. JS Array.prototype.sort is considered unreliable
+                                 */
+                                items: extensionConfig?.comparePlanningItems != null
+                                    ? group.events.sort(extensionConfig.comparePlanningItems)
```

Comment:
I wonder if we should use some memo here, or is it fast enough we shouldn't bother?